src/App.module.css

```diff
@@ -0,0 +1,55 @@
+html,
```

Comment:
it is better to put it in a separate file for reseting styles without a .module, for example, normalize.css